components/InfoList/index.tsx

```diff
@@ -5,7 +5,8 @@ const InfoList: React.FC<{
   list: Array<{ field: string; content: React.ReactNode; expandable?: boolean }>
   style?: React.CSSProperties
   type?: 'one-column' | 'two-columns'
-}> = ({ title, list, style, type = 'one-column' }) => {
+  titleSuffix?: Array<React.ReactNode>
```

Comment:
`title` could be a react element that includes `title | suffix`, it would be more flexible cuz its internal content is customizable.